proto/frequenz/api/reporting/v1/reporting.proto

```diff
@@ -296,6 +296,21 @@ message ListMicrogridComponentsDataResponse {
 //     like errors or operational states of the components during the specified time period.
 //
 message ReceiveMicrogridComponentsDataStreamRequest {
+  // Filter for sources for metrics, specified for a single metric at a time.
+  // Specifying a metric here will only stream data for that metric where data
+  // can be found tagged with the given source names.
+  // This is optional, and if omitted, the default metric will be used to stream data.
+  // If a metric is given here but not in the `metrics` field below,
+  // the metric source info will simply be ignored.
```

Comment:
I feel that these lines are more apt for the line `repeated MetricSourceOptions metric_sources = 4;`, and not here, because these lines relate to how the `MetricSourceOptions` message is used. What do you think?